tests/unit/lms/services/h_api_test.py

```diff
@@ -16,42 +15,39 @@
 from lms.services.h_api import HAPI
 from lms.values import HUser
 
-BASE_URL = "http://example.com/base/"
-AUTHORITY = "test-authority"
```

Comment:
The tests now just use the global `pyramid_request` fixture, instead of creating their own `p_request` fixture and adding these constants to it, so these constants are no longer needed. Also good to remove constants floating around at the top of test files.